webpack.config.babel.js

```diff
@@ -163,6 +163,7 @@ module.exports = ({ stats = false } = {}) => ({
       MOCK_WALLET: process.env.MOCK || 'false',
       MOCK_TOKEN_LIST: process.env.MOCK || 'false',
       MOCK_ERC20: process.env.MOCK || 'false',
+      MOCK_WETH: process.env.MOCK_WETH || 'false',
```

Comment:
Nope, wrong var. It already compares against `MOCK_WETH`. You should compare against `MOCK`.

```suggestion
      MOCK_WETH: process.env.MOCK || 'false',
```

Reply:
You'll also need to add it to `test/globalSetup.ts`

Reply:
Right! changing